app/services/session_encryptor.rb

```diff
@@ -1,3 +1,5 @@
+require 'user_access_key'
```

Comment:
I was trying to solve an unrelated bug and that crept into this PR. The problem is that the rails dev server autorestart has a problem with the encrypted session serializer. Not sure that this fixes it so I will remove.